### PR TITLE
feat: add metadata to node secrets and functions to generate it

### PR DESCRIPTION
**Pull request by fkondej**
closes vegaprotocol/devopsscripts#75

### CODE REVIEW

File: generate/name.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	"fmt"
+	"math/big"
+	"net/http"
+	"time"
+)
+
+var (
+	adjectives = []string{
+		"bright", "colossal", "condescending", "confused", "cooperative", "costly", "courageous",
+		"flabby", "glamorous", "gorgeous", "handsome", "long", "magnificent", "muscular", "plain", "plump",
```
Comment (edd) [inline generate/name.go:14]:
```suggestion
		"oppulent", "magnificent",
```

File: generate/name.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		"ashy", "blue", "gray", "green", "icy", "lemon", "mango", "orange", "purple", "red", "salmon",
+		"important", "inexpensive", "mealy", "odd", "poor", "powerful", "rich", "shy", "tender",
```
Comment (edd) [inline generate/name.go:16]:
```suggestion
		"important", "inexpensive", "fluky", "odd", "promising", "powerful", "rich", "shy", "tender",
```

File: generate/name.go [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+		"unimportant", "uninterested", "vast", "wrong",
+		"crashing", "echoing", "faint", "harsh", "howling", "loud", "melodic", "noisy",
+		"purring", "quiet", "rhythmic", "thundering", "wailing", "whining", "whispering",
+		"adorable", "adventurous", "agitated", "alert", "bored", "brave",
+		"determined", "distressed", "disturbed", "dizzy",
+		"excited", "extensive", "exuberant", "frustrating", "funny", "fuzzy",
+		"hungry", "icy", "ideal", "immense", "impressionable", "intrigued", "irate", "foolish", "frantic", "fresh",
+		"attractive", "bald", "beautiful", "clean", "dazzling", "elegant", "fancy", "fit",
```
Comment (edd) [inline generate/name.go:24]:
```suggestion
		"clean",
```

File: generate/name.go [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+		"graceful", "greasy", "grieving", "gritty", "grotesque", "grubby", "grumpy", "handsome", "happy", "hollow",
```
Comment (edd) [inline generate/name.go:25]:
```suggestion
		"graceful", "gritty", "happy", "hollow",
```

File: generate/name.go [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+		"quaint", "scruffy", "shapely", "short", "skinny", "stocky", "ugly", "unkempt", "unsightly",
+		"friendly", "frightened", "glorious", "gorgeous", "happy", "harebrained", "healthy",
```
Comment (edd) [inline generate/name.go:27]:
```suggestion
		"friendly", "frightened", "glorious", "happy", "harebrained", "healthy",
```

File: generate/name.go [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+		"helpful", "helpless", "high", "hollow", "homely", "large", "lazy", "livid", "lonely", "loose", "lovely", "lucky",
```
Comment (edd) [inline generate/name.go:28]:
```suggestion
		"helpful", "helpless", "high", "hollow", "loose", "lovely", "lucky",
```

File: generate/name.go [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+		"alive", "better", "careful", "clever", "dead", "easy", "famous", "gifted", "hallowed", "helpful",
+		"mysterious", "narrow", "nasty", "outrageous", "panicky", "perfect", "perplexed", "quizzical", "teeny", "tender", "tense",
```
Comment (edd) [inline generate/name.go:30]:
```suggestion
		"mysterious", "narrow", "perfect", "perplexed", "quizzical", "tender", "tense",
```

File: generate/name.go [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+		"terrible", "tricky", "troubled", "unsightly", "upset", "wicked", "yummy", "zippy",
```
Comment (edd) [inline generate/name.go:31]:
```suggestion
		"terrible", "tricky", "wicked", "yummy", "zippy",
```

File: generate/name.go [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+	}
+	nouns = []string{
+		"table", "chair", "vacuum", "man", "boy", "river", "cloud", "beach", "party", "ball", "bat",
+		"book", "pen", "pencil", "cookie", "biscuit", "butterfly", "bee", "hill", "baby", "pool",
+		"apple", "mango", "day", "week", "month", "flowers", "honey", "train", "bus", "car", "hat", "shirt", "sister",
+		"brother", "father", "milk", "coffee", "bed", "plant", "tree", "horse", "wall", "cat", "monkey",
+		"dog", "roof", "king", "queen", "shoe", "park", "bird", "pond", "duck", "farmer", "cow", "computer", "fan", "television",
```
Comment (edd) [inline generate/name.go:38]:
```suggestion
		"dog", "roof", "chimney", "tile", "shoe", "park", "bird", "pond", "duck", "farmer", "sheep", "computer", "fan", "television",
```

File: generate/name.go [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+	"time"
+)
+
+var (
+	adjectives = []string{
+		"bright", "colossal", "condescending", "confused", "cooperative", "costly", "courageous",
+		"flabby", "glamorous", "gorgeous", "handsome", "long", "magnificent", "muscular", "plain", "plump",
+		"ashy", "blue", "gray", "green", "icy", "lemon", "mango", "orange", "purple", "red", "salmon",
+		"important", "inexpensive", "mealy", "odd", "poor", "powerful", "rich", "shy", "tender",
+		"unimportant", "uninterested", "vast", "wrong",
```
Comment (edd) [inline generate/name.go:17]:
```suggestion
		"opportune", "timely", "vast", "wrong",
```

**Review by edd — COMMENTED** [review]
I've madea few suggestions to the adjectives/nouns set that works around some mildly weird combos that could have appeared (e.g. `poor cow`, `flabby king`). Totally non-essential edits.

**Review by mkjmdski — APPROVED** [review]

**Pull request merged**